packages/components/src/components/contract-card/contract-card-items/tick-counter-bar.tsx

```diff
@@ -9,7 +9,7 @@ type TTickCounterBar = {
 const TickCounterBar = ({ current_tick, label, max_ticks_duration }: TTickCounterBar) => (
     <div className='dc-tick-counter-bar__container'>
         <div className='dc-tick-counter-bar__track'>
-            <Text size='xxs' weight='bold' className='dc-tick-counter-bar__text'>
+            <Text size='xxs' weight='bold' align='center' className='dc-tick-counter-bar__text'>
```

Comment:
hi @henry-deriv :) The purpose of the PR is to replace an FE-calculated value with a BE-provided value. why do we need to change this alignment?